bucket/tagspaces.json

```
@@ -0,0 +1,46 @@
+{
+    "version": "3.2.1",
+    "description": "An offline document manager with tagging support.",
+    "homepage": "https://www.tagspaces.org",
+    "license": "AGPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/tagspaces/tagspaces/releases/download/v3.2.1/tagspaces-win-3.2.1.exe#/dl.7z",
+            "hash": "00159f6a4ea11578a0b4d14bd2502e4cda76071c5b8917d6460377d08ecdff40",
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Force -Recurse"
```

No need to repeat $dir with escaped quotes
```suggestion
                    "Remove-Item \"$dir\\*\" -Include '$PLUGINSDIR', 'Uninstall*' -Force -Recurse"
```